modules/install-upgrade/partials/minimum-sizing.adoc

```diff
@@ -5,67 +5,51 @@
 |Node Count
 |Note
 
-.2+|DEV
+|SANDBOX (STANDALONE)
+|(combined) Zookeeper, Broker, and Bookie
+|1
+|
+
+.2+|DEVELOPMENT
 |(combined) Zookeeper, Broker, and Bookie
 |3
-|1 node per AZ +
-In the simplest case, Pulsar can run on only 1 server host as a standalone mode.
-|Admin Console, and Heartbeat
+|1 node per AZ
+|Admin Console and Heartbeat
+|1
+|Shared, 1 server host; placed in any rack
+
+.4+|SINGLE REGION PRODUCTION ENVIRONMENT (1 region)
+|Zookeeper
+|3
+|
+|Broker
+|3
+|
+|Bookie
+|3
+|
+|Admin Console and Heartbeat
 |1
 |Shared 1 server host; placed in any rack
 
-.4+|TEST
+.6+|HIGH-AVAILABILITY PRODUCTION ENVIRONMENT (2 region)
 |Zookeeper
 |3
 |1 node per AZ
-| (Combined) Broker and Function Worker
-| 3
-| 1 node per AZ +
-Even if the Pulsar function is needed, it runs as part of the brokers. No dedicated function worker node is needed.
-| Bookie
-| 6
-| 2 nodes per AZ +
-See note 3 below
-| Admin Console and Heartbeat
-| 1
-| Shared 1 server host; placed in any rack
-
-.6+|PROD
-| Zookeeper
-| 3
-| 1 node per AZ
-| Broker
-| 3
-| 1 node per AZ (separate physical racks)
-| (Dedicated) Function Worker*
-| 6
-| 2 node per AZ +
+|Broker
+|3
+|1 node per AZ (separate physical racks)
+|(Dedicated) Function Worker*
+|6
+|2 node per AZ +
 This is optional and only needed when Pulsar functions are required.
```

Comment:
Also please make a note that the number of function worker nodes really depends on how many functions would be deployed and how heavy the workload is for each function